fivejars/openy_gated_content
# [PRODDEV-479] [PRODDEV-480]  Flag new shared content

Do not merge PR, only review it and approve/request changes.
Merge is allowed by QA Engineer only.

**Related Issue/Ticket:**

**PLEASE CHECK BASE BRANCH FOR YOUR PR**
**ONLY urgent and approved fixes for point release should go to master branch**

https://openy.atlassian.net/browse/PRODDEV-479
https://openy.atlassian.net/browse/PRODDEV-480

## Steps to test:

You will need to have a site connected to a shared content server in order to test and have the shared content server all configured.

Check out this video: 

https://user-images.githubusercontent.com/238201/145146512-354e3c73-8feb-47c8-98cc-c45ae9312ccd.mp4


- [ ] Observe "Fetch to my site" button is duplicated on the top of the form.
![Fetch_shared_content___Drush_Site-Install](https://user-images.githubusercontent.com/238201/144768539-093f1559-20f3-4b95-adb0-584bdf2792e2.png)
- [ ] Observe top "Fetch to my site" button works exactly the same as the bottom button.

- [ ] Observe once content is added to a site, the checkbox and button are disabled and the button text changes from "Preview" to "Added"
![Fetch_shared_content___Drush_Site-Install](https://user-images.githubusercontent.com/238201/144768594-ccbc0f62-3bc2-4bca-9cb4-4e2fa62c751a.png)

- [ ] Observe when you click "Preview" a "Fetch to my site" button that fetches content when clicked.
- [ ] Observe close button that closes the preview with no action.

![Fetch_shared_content___Drush_Site-Install](https://user-images.githubusercontent.com/238201/144768624-5ed728ce-97be-4841-8cbb-e6ee06dcd408.png)

- [ ] Observe content is bolded when new (or when it's your first session)
- [ ] Observe content is unbolded when you Preview it and then close the modal.
- [ ] Observe content is unbolded when you start a new session.

## Quality checks:

Please check these boxes to confirm this PR covers the following cases:

- Maintaining our upgrade path is essential. Check one or the other:
- Front end fixes should be tested against all of the Open Y Themes.
  team in Slack
